airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/models/ci_report.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass(frozen=True)
+class ConnectorNightlyReport:
+    file_path: str
+    pipeline_name: Optional[str] = None
+    run_timestamp: Optional[str] = None
+    run_duration: Optional[float] = None
+    success: Optional[bool] = None
+    failed_steps: Optional[list] = None
+    successful_steps: Optional[list] = None
+    skipped_steps: Optional[list] = None
+    gha_workflow_run_url: Optional[str] = None
+    pipeline_start_timestamp: Optional[int] = None
+    pipeline_end_timestamp: Optional[int] = None
+    pipeline_duration: Optional[int] = None
+    git_branch: Optional[str] = None
+    git_revision: Optional[str] = None
+    ci_context: Optional[str] = None
```

Comment:
nit: Would be cool to import the types from the `pipelines` python package and deserialize to the CIContext enum.

Reply:
Agreed! Ill add a TODO here as that will be easier to do once pipelines is in the airbyte-ci folder